### PR TITLE
feat(stable): add templates

### PR DESCRIPTION
**Pull request by kdada**
**What this PR does**
- feat(stable): add mongo 3.6.1
- feat(stable): add mysql 5.7.20
- feat(stable): add mariadb 10.2.12
- feat(stable): add influxdb 1.4.2
- feat(stable): add filemanager 1.4.4
- feat(stable): add wordpress 4.9.1
- feat(stable): add redmine 3.4.4

**why we need it**

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by zoumo** [comment]
add chart version to commits

File: stable/mysql/5.7.20/values.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        value: user
+      - name: MYSQL_PASSWORD
+        value: password
+      mounts:
+      - path: "/var/lib/mysql"
+        name: data
+      command: []
+      args: []
+      imagePullPolicy: Always
+      image: cargo.caicloudprivatetest.com/library/mysql:5.7.20
```
Comment (ddysher) [inline stable/mysql/5.7.20/values.yaml]: How do we manage different image repo?

Comment (kdada) [reply stable/mysql/5.7.20/values.yaml]: https://github.com/caicloud/charts/blob/master/build/charts/build.sh#L36

Build script would replace it.

Comment (zoumo) [reply stable/mysql/5.7.20/values.yaml]: images in library are synced from upstream like docker hub.

Comment (ddysher) [reply stable/mysql/5.7.20/values.yaml]: OK

**Pull request merged**